src/include/parser/expression/comparison_expression.h

```diff
@@ -0,0 +1,38 @@
+#pragma once
+
+#include <memory>
+#include <utility>
+#include <vector>
+#include "parser/expression/abstract_expression.h"
+#include "parser/expression/expression_defs.h"
+#include "type/type_id.h"
+
+namespace terrier {
+namespace parser {
+namespace expression {
```

Comment:
Do we want a third-level namespace? @tli2

Reply:
I think because codegen have them, it's fine.

However, I do not like the use of third-level namespace here. Adding `expression::` to every expression doesn't tell people anything they don't know and is just code clutter. Consider `expression::Comparison` or just drop the namespace and use `ComparisonExpression`.